src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -943,14 +933,18 @@ public void getModifiedDocIds(DocIdPusher pusher)
           PreparedStatement stmt = getUpdateStreamFromDb(conn);
           ResultSet rs = stmt.executeQuery();
           BufferedPusher outstream = new BufferedPusher(pusher)) {
-        hasTimestamp =
-            hasColumn(rs.getMetaData(), GsaSpecialColumns.GSA_TIMESTAMP);
+        ResultSetMetaData rsmd = rs.getMetaData();
+        boolean hasAction = hasColumn(rsmd, GsaSpecialColumns.GSA_ACTION);
+        log.log(Level.FINEST, "hasAction: {0}", hasAction);
+        hasTimestamp = hasColumn(rsmd, GsaSpecialColumns.GSA_TIMESTAMP);
         log.log(Level.FINEST, "hasTimestamp: {0}", hasTimestamp);
```

Comment:
I see, this was copied from here. I would change this, too, for a better log message (one someone might be able to understand and fix their config based on).